src/main/resources/org/terasology/launcher/packages/subschema/TrackedPackage.json

```diff
@@ -0,0 +1,15 @@
+{
+  "4.0": {
+    "type": "string",
+    "minLength": 1
+  },
+
+  "4.1": {
+    "type": "object",
+    "properties": {
+      "id": {"type": "string", "minLength": 1},
+      "name": {"type": "string", "minLength": 1}
+    },
+    "required": ["id", "name"]
+  }
```

Comment:
As mapping the schemes to specific launcher releases is a bit hard already with the two we have, let's just number them consecutively.

Reply:
Could go for a simple 'v1', 'v2', ... scheme then :+1: